core/scripts/download-chrome.sh

```diff
@@ -33,8 +45,7 @@ else
   exit 1
 fi
 
-if [ -e "$CHROME_PATH" ]; then
-  echo "cached chrome found"
-else
-  curl "$url" -Lo chrome.zip && unzip -q chrome.zip
-fi
+mkdir -p .tmp-download && cd .tmp-download
+curl "$url" -Lo chrome.zip && unzip -q chrome.zip && rm chrome.zip
```

Comment:
i'd probably just put all these on their own lines
conceivably the -e is equivalent to stringing everything with `&&` but i'm sure there's some idiosyncracies that I don't care about.  but.. its probably close enough

also no big whoop. i'm fine with &&'s too.